app/forms/curation_concerns/forms/work_form.rb

```diff
@@ -0,0 +1,83 @@
+module CurationConcerns
+  module Forms
+    class WorkForm
```

Comment:
Then this should be a new class that extends CurationConcerns::Forms::WorkForm, rather than overwrites it.